src/components/ProgressSteps/ProgressStep.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
+import React from 'react';
+import checkmark from '../../assets/icons/checkmark.svg';
+import Loader from '../Loader/Loader';
+
+type ProgressStepProps = {
+  value: string;
+  description: string;
+  isCompleted?: boolean;
+};
+
+function ProgressStep({
+  value,
+  description,
+  isCompleted = false,
+}: ProgressStepProps): JSX.Element {
+  return (
+    <ProgressStepStyled>
+      <div>
+        <ProgressStepState>{value}</ProgressStepState>
+        <ProgressStepText>{description}</ProgressStepText>
+      </div>
+      <ProgressStepIndicator>
+        {isCompleted ? <img src={checkmark} alt={'progress icon'} /> : <Loader />}
```

Comment:
no need to use JSX for static strings
```suggestion
        {isCompleted ? <img src={checkmark} alt="progress icon" /> : <Loader />}
```